packages/api-catalog-service/src/graph/namespace/updateSearch.ts

```diff
@@ -0,0 +1,62 @@
+import { fetch } from 'undici';
```

Comment:
Any specific reason why you're using `undici`?
Use `cross-fetch` instead. It has a significantly lesses package size.

Reply:
undici was already used over there with many placed. so I reused it

Reply:
Undici is used mainly for apollo-http data source. 

Also undici is more than just a fetch. It does lot of things caching and concurrent request etc.

Reply:
Oh ohk. I thought it was added for just making fetch requests. If it's used as apollo data source, then it's fine.